lib/index.js

```diff
@@ -368,6 +348,11 @@ exports.init = function(options, callback) {
                 }
             }
         });
+
+        if (!Object.keys(sorted).length) {
```

Comment:
Unrelated drive-by, moved this up to sit after sorting and before an license exclusion / filtering.